### PR TITLE
Refactor platform with Module feature

### PR DESCRIPTION
**Pull request by lohanidamodar**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -17,8 +17,9 @@
         "php": ">=8.0",
         "ext-json": "*",
         "ext-redis": "*",
-        "utopia-php/framework": "0.*.*",
-        "utopia-php/cli": "0.15.*"
+        "utopia-php/framework": "0.33.*",
+        "utopia-php/cli": "0.15.*",
+        "utopia-php/queue": "0.7.*"
```
Comment (eldadfux) [inline composer.json:22]: Next step would also be to add realtime here. We can do it when we start using utopia/di in all libraries.

File: src/Platform/Platform.php [hunk 2]
```diff
@@ -114,24 +134,24 @@ protected function initHttp(): void
      *
      * @return void
      */
-    protected function initCLI(): void
+    protected function initCLI(array $services): void
```
Comment (eldadfux) [inline src/Platform/Platform.php:137]: Let's use tasks instead of CLI

**Review by eldadfux — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (lohanidamodar) [reply composer.json:22]: Sure, but that will take time

File: src/Platform/Platform.php (hunk 2 above)
Comment (lohanidamodar) [reply src/Platform/Platform.php:137]: done

File: composer.json (hunk 1 above)
Comment (eldadfux) [reply composer.json:22]: NP, I have a POC being tested.

**Review by eldadfux — APPROVED** [review]

**Pull request merged**